FE/src/App.js

```diff
@@ -1,25 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
-import { ThemeProvider } from 'styled-components';
+// import { ThemeProvider } from 'styled-components';
 
 import { Header } from './components/Header';
+import { Auth } from './pages/Auth';
+import { IssueDetail } from './pages/IssueDetail';
 import { IssueList } from './pages/IssueList';
+import { LabelList } from './pages/LabelList';
+import { Login } from './pages/Login';
+import { MilestoneList } from './pages/MilestoneList';
 import { NewIssue } from './pages/NewIssue';
 import { GlobalStyle } from './styles/GlobalStyle';
-import { lightTheme } from './styles/theme';
+// import { lightTheme } from './styles/theme';
 
 const App = () => {
+  const [login, setLogin] = useState({ isLogin: false, accessToken: '' });
   return (
     <>
       {/* <ThemeProvider theme={lightTheme}> */}
       <GlobalStyle />
       <BrowserRouter>
         <div className="App">
-          {/* TODO : user 로그인 하면 헤더 고정 시키기 <Header /> */}
+          {/* {login.isLogin && <Header />} */}
+          {<Header />}
           <Routes>
-            <Route path="/" element={<IssueList />} />
+            <Route path="/" element={<Login />} />
+            <Route path="/auth" element={<Auth />} />
+            <Route path="/issues" element={<IssueList />} />
             <Route path="/newIssue" element={<NewIssue />} />
+            <Route path="/issueDetail/:id" element={<IssueDetail />} />
```

Comment:
p4)
뒤에 id가 붙기 때문에 detail이라는 단어는 없어도 괜찮습니다. 😀
```suggestion
            <Route path="/issues/:id" element={<IssueDetail />} />
```

Reply:
이슈 작성하면 바로 해당 아이디 페이지로 넘어가니까 detail 페이지가 따로 필요하지 않겠네요 ! 참고해서 로직 수정해보겠습니다 :)